### ISSUE TITLE
Oliver Evans's Mill Was Not Steam Operated

### ISSUE THREAD

**Issue by tbpassin** [hendricius/the-sourdough-framework#383]
Some of Oliver Evan's machine inventions were powered by steam, but they came after publication of his book about his grain mill ideas.  All restored Evans mills I've seen were powered by water wheels.

**Comment by cedounet** [comment]
That is correct… will change the text, he still improved a lot or at least claims it in his book.

Thanks
